lib/modules/uscore_v3.1.0/bp_sequence.rb

```diff
@@ -630,157 +482,85 @@ def validate_resource_item(resource, property, value)
-          'Observation.component.valueDateTime.code',
-          'Observation.component.valuePeriod.code',
-          'Observation.component.dataAbsentReason'
+          { path: 'Observation.status', fixed_value: '' },
+          { path: 'Observation.category', fixed_value: '' },
+          { path: 'Observation.category.coding', fixed_value: '' },
+          { path: 'Observation.category.coding.system', fixed_value: 'http://terminology.hl7.org/CodeSystem/observation-category' },
+          { path: 'Observation.category.coding.code', fixed_value: 'vital-signs' },
+          { path: 'Observation.code', fixed_value: '' },
+          { path: 'Observation.subject', fixed_value: '' },
+          { path: 'Observation.effectiveDateTime', fixed_value: '' },
+          { path: 'Observation.effectivePeriod', fixed_value: '' },
+          { path: 'Observation.valueQuantity', fixed_value: '' },
+          { path: 'Observation.dataAbsentReason', fixed_value: '' },
+          { path: 'Observation.component.valueQuantity', fixed_value: '' },
+          { path: 'Observation.component.valueCodeableConcept', fixed_value: '' },
+          { path: 'Observation.component.valueString', fixed_value: '' },
+          { path: 'Observation.component.valueBoolean', fixed_value: '' },
+          { path: 'Observation.component.valueInteger', fixed_value: '' },
+          { path: 'Observation.component.valueRange', fixed_value: '' },
+          { path: 'Observation.component.valueRatio', fixed_value: '' },
+          { path: 'Observation.component.valueSampledData', fixed_value: '' },
+          { path: 'Observation.component.valueTime', fixed_value: '' },
+          { path: 'Observation.component.valueDateTime', fixed_value: '' },
+          { path: 'Observation.component.valuePeriod', fixed_value: '' },
+          { path: 'Observation.component.valueQuantity.value', fixed_value: '' },
+          { path: 'Observation.component.valueCodeableConcept.value', fixed_value: '' },
+          { path: 'Observation.component.valueString.value', fixed_value: '' },
+          { path: 'Observation.component.valueBoolean.value', fixed_value: '' },
+          { path: 'Observation.component.valueInteger.value', fixed_value: '' },
+          { path: 'Observation.component.valueRange.value', fixed_value: '' },
+          { path: 'Observation.component.valueRatio.value', fixed_value: '' },
+          { path: 'Observation.component.valueSampledData.value', fixed_value: '' },
+          { path: 'Observation.component.valueTime.value', fixed_value: '' },
+          { path: 'Observation.component.valueDateTime.value', fixed_value: '' },
+          { path: 'Observation.component.valuePeriod.value', fixed_value: '' },
+          { path: 'Observation.component.valueQuantity.unit', fixed_value: '' },
+          { path: 'Observation.component.valueCodeableConcept.unit', fixed_value: '' },
+          { path: 'Observation.component.valueString.unit', fixed_value: '' },
```

Comment:
These `value`, `unit`, `system`, and `code` must supports are only for the required slices in this profile which limit `value[x]` to `valueQuantity`.

Reply:
Fixed by checking for any value[x]. I think profile validation would be able to check that it's valueQuantity specifically.